repositories.yaml

```diff
@@ -1,502 +1,471 @@
+      - org.mozilla.components:places
+    channels:
+      - v1_name: firefox-android-release
+        app_id: org.mozilla.firefox
+        app_channel: release
+        additional_dependencies:
+          - org.mozilla.components:browser-engine-gecko-beta
+        description: >
+          Release channel of Firefox for Android.
+      - v1_name: firefox-android-beta
+        app_id: org-mozilla-firefox-beta
+        app_channel: beta
+        additional_dependencies:
+          - org.mozilla.components:browser-engine-gecko-beta
+        description: >
+          Beta channel of Firefox for Android.
+      - v1_name: fenix
+        app_id: org.mozilla.fenix
+        app_channel: nightly
+        additional_dependencies:
+          - org.mozilla.components:browser-engine-gecko-beta
+        description: >
+          Nightly channel of Firefox for Android.
+          Prior to June 2020, this app_id was used for the beta channel
+          of Firefox Preview.
+      - v1_name: fenix-nightly
+        app_id: org.mozilla.fenix.nightly
+        app_channel: nightly
+        deprecated: true
+        additional_dependencies:
+          - org.mozilla.components:browser-engine-gecko-nightly
+        description: >
+          Nightly channel of Firefox Preview.
+      - v1_name: firefox-android-nightly
+        app_id: org.mozilla.fennec.aurora
+        app_channel: nightly
+        deprecated: true
+        additional_dependencies:
+          - org.mozilla.components:browser-engine-gecko-beta
+        description: >
+          Nightly channel of Firefox for Android users migrated to Fenix;
+          delisted in June 2020.
+  - app_name: firefox_ios
+    canonical_app_name: Firefox for iOS
+    description: Firefox for iOS
+    notification_emails:
+      - tlong@mozilla.com
+      - frank@mozilla.com
+    url: https://github.com/mozilla-mobile/firefox-ios
+    metrics_files:
+      - Client/metrics.yaml
+    branch: main
+    dependencies:
+      - org.mozilla.deprecated:glean
+    channels:
+      - v1_name: firefox-ios-release
+        app_id: org.mozilla.ios.Firefox
+        app_channel: release
+      - v1_name: firefox-ios-beta
+        app_id: org.mozilla.ios.FirefoxBeta
+        app_channel: beta
+      - v1_name: firefox-ios-dev
+        app_id: org.mozilla.ios.Fennec
+        app_channel: nightly
+  - app_name: reference_browser
+    canonical_app_name: Reference Browser
+    description: >
+      A full-featured browser reference implementation using Mozilla Android
+      Components
+    notification_emails:
+      - frank@mozilla.com
+      - mdroettboom@mozilla.com
+    url: https://github.com/mozilla-mobile/reference-browser
+    prototype: true
+    metrics_files:
+      - app/metrics.yaml
+    dependencies:
+      - org.mozilla.components:service-glean
+      - org.mozilla.components:lib-crash
+    channels:
+      - v1_name: reference-browser
+        app_id: org-mozilla-reference-browser
```

Comment:
drive-by: I think this should be `org.mozilla.reference.browser`; cf https://github.com/mozilla-mobile/reference-browser/blob/349b13324f35bf6357af4a07ae875cc58aec7753/app/src/main/AndroidManifest.xml#L7